src/main/server/enable_ssl.sh

```diff
@@ -279,36 +279,12 @@ auth_tomcat(){
 
 create_cron_job(){
 
-	#add renew job to crontab
-	$SUDO crontab -l > mycron
-	
-	
-	#echo new cron into cron file
-	#run renew script in each 85 days
-	$SUDO echo "00 03 */85 * * cd $INSTALL_DIRECTORY && ./enable_ssl.sh -d $domain -r" >> mycron
-	
-	
-	OUT=$?
-	if [ $OUT -ne 0 ]; then
-	 echo -e $ERROR_MESSAGE
-	 exit $OUT
-	fi
-
-
-	#install new cron file
-	$SUDO crontab mycron
-	
-	OUT=$?
-	if [ $OUT -ne 0 ]; then
-	 echo -e $ERROR_MESSAGE
-	 exit $OUT
-	fi
-
-	#remove temp cron
-	$SUDO rm mycron
-	
-	#restart cron jobs
-	$SUDO systemctl restart cron
+	#Check if file does not exist
+        if [ ! -f $cronfile ]; then
```

Comment:
Are we sure that `$cronfile` variable exist?

Reply:
fixed.